### PR TITLE
Fix incorrect scrolling when `Wrap` is set to `true`

### PR DESCRIPTION
**Pull request by dankox**
Fprint into the View doesn't scroll correctly content if `Autoscroll` and `Wrap` are turned on.

It's caused by using incorrect slice when checking length of lines if the content should be scrolled and how much.

There is new test for this called `wrap_autorscroll.go` which adds content to the view when hitting `Tab` key.

### CODE REVIEW

File: _examples/wrap_autoscroll.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
+// Copyright 2014 The gocui Authors. All rights reserved.
```
Comment (mjarkk) [inline _examples/wrap_autoscroll.go]: Do we need this extra example?  
This seems more like a test for re-creating locally a problem.

Comment (dankox) [reply _examples/wrap_autoscroll.go]: Yeah, you are right.... It kinda simulates the idea of autoscroll and wrap together.     

To be honest, you can recreate that by using `active.go` or `custom_frames.go` and makes your terminal small enough to force it to wrap the text in second view.    
So I can remove this exmple.

**Review by mjarkk — APPROVED** [review]

**Comment by jon4hz** [comment]
could you please make a new release with this fix?

**Comment by mjarkk** [comment]
Ofc, here you go [v1.0.1](https://github.com/awesome-gocui/gocui/releases/tag/v1.0.1)

**Pull request merged**